### PR TITLE
Add Note RE to Charming Liar

### PR DESCRIPTION
**Pull request by DocSchlock**

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -2054,6 +2054,9 @@
                 "Prompt": "Treat Cards as darts or daggers?"
             },
             "Feat": {
+                "CharmingLiar":{
```
Comment (simonward) [inline static/lang/re-en.json:2057]:
```suggestion
                "CharmingLiar": {
```

File: static/lang/re-en.json [hunk 2]
```diff
@@ -2054,6 +2054,9 @@
...
+                    "Note":"When you get a critical success using the Lie action, the target's attitude toward you improves by one step, as though you'd succeeded at using Diplomacy to Make an Impression."
```
Comment (simonward) [inline static/lang/re-en.json:2058]:
```suggestion
                    "Note": "When you get a critical success using the Lie action, the target's attitude toward you improves by one step, as though you'd succeeded at using Diplomacy to Make an Impression."
```

File: packs/feats/charming-liar.json [hunk 3]
```diff
@@ -28,7 +28,20 @@
...
+        "rules": [
+            {
+                "key": "Note",
+                "outcome": [
+                    "criticalSuccess"
+                ],
+                "predicate": [
+                    "action:lie"
+                ],
+                "selector": "skill-check",
```
Comment (simonward) [inline packs/feats/charming-liar.json:40]: Should the selector be `deception`?

Comment (DocSchlock) [reply packs/feats/charming-liar.json:40]: Strangely, Charming Liar doesn't mention the Lie actions it benefits are solely using Deception. If a PC had a way to replace Deception with another skill for the Lie Action, Charming Liar would still apply.

**Pull request merged**